src/bom.rs

```diff
@@ -74,7 +74,8 @@ impl<'a> Default for Bom<'a> {
             spec_version: "1.3",
             version: 1,
             serial_number: Uuid::new_v4(),
-            components: iter.into_iter().map(Component::library).collect(),
+            components: vec!(),
```

Comment:
I figured this was reasonable, for a Default impl anyways. Just makes sure there is a empty `Vec` more or less.